draft-ietf-quic-recovery.md

```diff
@@ -906,9 +906,26 @@ congestion window, which is recommended to be the minimum of
 max_datagram_size is the current maximum size of a datagram for the connection,
 not including UDP or IP overhead.
 
-As an example of a well-known and publicly available implementation of a flow
-pacer, implementers are referred to the Fair Queue packet scheduler (fq qdisc)
-in Linux (3.11 onwards).
+Endpoints can implement pacing as they choose, as long as bursts are not larger
+than the initial window and average sending rate over a round trip does not
+exceed a small multiple, `pacing_gain`, of the congestion window. Larger
+bursts can reduce CPU utilization at the risk of increased retransmits rates.
```

Comment:
This last sentence seems a little disconnected from the rest.  A small tweak might might help:

```suggestion
exceed a small multiple, `pacing_gain`, of the congestion window. Endpoints can
trade off sending packets in bursts, which might be done to reduce CPU
utilization, against even pacing, which has a lower risk of inducing packet
loss.
```